public/app/features/plugins/extensions/registry/ExposedComponentsRegistry.ts

```diff
@@ -16,10 +18,13 @@ export class ExposedComponentsRegistry extends Registry<
   ExposedComponentRegistryItem,
   PluginExtensionExposedComponentConfig
 > {
-  constructor(initialState: RegistryType<ExposedComponentRegistryItem> = {}) {
-    super({
-      initialState,
-    });
+  constructor(
+    options: {
+      registrySubject?: ReplaySubject<RegistryType<ExposedComponentRegistryItem>>;
+      initialState?: RegistryType<ExposedComponentRegistryItem>;
+    } = {}
+  ) {
+    super(options);
```

Comment:
Is there a reason this is not applied also to AddedLinkRegistry and AddedComponentsRegistry?

Reply:
(not tested, but think e2e tests will pass if you apply to all registries)

Reply:
Fixed in https://github.com/grafana/grafana/pull/92014/commits/7bc5a05d646e7c36c9cc7de56cc3c04010a88013.